layout/dir.proj

```diff
@@ -13,7 +13,7 @@
     <TargetFrameworkMonikers Include="netstandard1.5" />
     <TargetFrameworkMonikers Include="netstandard1.6" />
     <TargetFrameworkMonikers Include="netstandard1.7" />
-    <TargetFrameworkMonikers Include="netcoreapp1.0">
```

Comment:
Do we also want to keep netcoreapp1.0 around to be able to have a layout drop of what we have shipped already? this can come handy in comparing what we have added since.